vehicles/test/util/mocks/data/jwt.mock.ts

```diff
@@ -109,6 +110,7 @@ export const redCompanyCvClientUserJWTMock: IUserJWT = {
   orbcUserLastName: constants.RED_COMPANY_CVCLIENT_LAST_NAME,
   orbcUserAuthGroup: constants.RED_COMPANY_CVCLIENT_USER_AUTH_GROUP,
   orbcUserDirectory: constants.RED_COMPANY_CVCLIENT_USER_STATUS_DIRECOTRY,
+  clientId: null,
```

Comment:
I do not see any changes to IUserJWT object as part of the PR

Reply:
It was done before , added clientId

Reply:
okay